po/extra/lt.po

```diff
@@ -0,0 +1,302 @@
+msgid "FilesView: Properly center the dialog against the app window"
+msgstr ""
+"RinkmenosRodinys: Tinkamai centruoti dialogo langą priešais programos "
+"langelį"
+
+#: data/pinit.appdata.xml.in:102
+msgid "Restore last opened view and its state"
+msgstr "Grąžinti paskutinį naudotą rodinį ir jo būseną"
+
+#: data/pinit.appdata.xml.in:103
+msgid "Add new style switcher"
+msgstr "Pridėti naują stilių perjungiklį"
+
+#: data/pinit.appdata.xml.in:104
+msgid "Correct titlebar label when no app name specified"
+msgstr ""
+"Ištaisyti pavadinimo juostos labelį kuomet nepateikiamas programėlės vardas"
+
+#: data/pinit.appdata.xml.in:105
+msgid "Readable copies"
+msgstr "Perskaitomos kopijos"
+
+#: data/pinit.appdata.xml.in:116
+msgid "FilesView: Ellipsize if label texts are too long"
+msgstr "RinkmenosRodinys: Tritaškis jei labelis per ilgas"
+
+#: data/pinit.appdata.xml.in:117
+msgid "Tell accepted file formats in FileChooser"
+msgstr "Nusakyti priimtinus rinkmenų formatus Rinkmenų parinkėjui"
+
+#: data/pinit.appdata.xml.in:121
+msgid "Add Brazilian Porguguese translation (thanks to @DanielTolentino)"
+msgstr "Add Brazilian Porguguese translation (thanks to @DanielTolentino)"
+
+#: data/pinit.appdata.xml.in:129
+msgid "EditView: Focus on file name entry by default"
+msgstr ""
+"KeitimoRodinys: pagal nutylėjimą susikoncentruoti ties rinkmenos įrašo "
+"įvestimi"
+
+#: data/pinit.appdata.xml.in:130
+msgid ""
+"FilesView: Make scrollable and fix the window get bigger if many entries"
+msgstr ""
+"RinkmenosRodinys: Įgyvendinta slankjuostė ir tvarkingas langelio dydis "
+"perpildant daugybe įvesčių"
+
+#: data/pinit.appdata.xml.in:134
+msgid "Add French translation (thanks to @NathanBnm)"
+msgstr "Add French translation (thanks to @NathanBnm)"
+
+#: data/pinit.appdata.xml.in:142
+msgid "Improved support for other desktop environments"
+msgstr "Patobulintas kitų darbalaukio aplinkų palaikymas"
+
+#: data/pinit.appdata.xml.in:143
+msgid "Add execution permisson on clicking the \"Pin It!\" button"
+msgstr "Pridėti vykdomąsias teises \"Pakišk tai!\" mygtuko paspaudimui"
+
+#: data/pinit.appdata.xml.in:154
+msgid "Add CategoryChooser"
+msgstr "Pridėti kategorijos parinkėją"
+
+#: data/pinit.appdata.xml.in:155
+msgid "Support Alt+Home to show welcome"
+msgstr "Palaikoma ALT+HOME pasisveikinimo vaizdavimui"
+
+#: data/pinit.appdata.xml.in:156
+msgid "Allow GPU acceralation by default"
+msgstr ""
+"Pagal nutylėjimą palaikomas Grafinio Procesorinio Vieneto akceleracija"
+
+#: data/pinit.appdata.xml.in:167
+msgid "Add 128px app icon (thanks to @hanaral)"
+msgstr "Add 128px app icon (thanks to @hanaral)"
+
+#: data/pinit.appdata.xml.in:168
+msgid "Rounded window corners on any stylesheet (thanks to @JeysonFlores)"
+msgstr "Rounded window corners on any stylesheet (thanks to @JeysonFlores)"
+
+#: data/pinit.appdata.xml.in:169
+msgid "Support multitouch gestures (thanks to @JeysonFlores)"
+msgstr "Support multitouch gestures (thanks to @JeysonFlores)"
```

Comment:
[Question]
You seem to leave some strings in this pull request as it is like here, but is there any reason for that?

Reply:
Translator credits goes in to the lines of 305 and 306.
I'm suggesting for You to revise the Idea to thank programmers in similar way as created for translators in 305 and 306, and maybe link thanks with Your provided messages 264-273 by adding some kind numerical value maybe.
This information is limited in necessity for the App Users, what a slacker I am.

Reply:
The "translator-credit" is a common way to credit translators of the **currently displaying language**. For example, the translation against "translator-credit" in the Polish translation should be only "PolPolyLingo", because you're the first and the only person who translated the app into Polish at this time. Other translators in other languages should be credited in the translation file for their language.

And these sentences like in 264-273 is not only credit the contributors but also let app users knows what changed when the app updated.

For these reasons I'm not mean to use the same method to credit translators and programmers.

Reply:
Let us leave this untranslated in English as it is.